src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/12.diff.sql

```diff
@@ -0,0 +1,56 @@
+/*************************************************************
+    Stored procedures for search parameter information
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     GetSearchParamStatuses
+--
+-- DESCRIPTION
+--     Gets all the search parameters and their statuses.
+--
+-- RETURN VALUE
+--     The search parameters and their statuses.
+--
+ALTER PROCEDURE dbo.GetSearchParamStatuses
+AS
+    SET NOCOUNT ON
+
+    SELECT SearchParamId, Uri, Status, LastUpdated, IsPartiallySupported FROM dbo.SearchParam
+GO
+
+--
+-- STORED PROCEDURE
+--     UpsertSearchParams
+--
+-- DESCRIPTION
+--     Given a set of search parameters, creates or updates the parameters.
+--
+-- PARAMETERS
+--     @searchParams
+--         * The updated existing search parameters or the new search parameters
+--
+ALTER PROCEDURE dbo.UpsertSearchParams
```

Comment:
>UpsertSearchParams [](http://example.com/codeflow?start=20&length=18)

@summaryOfChanges is no longer needed?

Reply:
It looks like we won't need to make this fix now that we are keeping `dbo.UpsertSearchParams` as is. But good catch, thanks!